Testscripts/Windows/PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1

```diff
@@ -246,10 +264,13 @@ collect_VM_properties
                     $resultMap["DatagramLoss"] = $($Line[3])
                 } else {
                     $resultMap["Throughput_Gbps"] = $($Line[1])
+                    $resultMap["SenderCyclesPerByte"] = $($Line[2])
                     $resultMap["Latency_ms"] = $($Line[3])
                     $resultMap["TXpackets"] = $($Line[4])
                     $resultMap["RXpackets"] = $($Line[5])
                     $resultMap["PktsInterrupts"] = $($Line[6])
+                    $resultMap["ConCreatedTime_us"] = $($Line[7])
```

Comment:
Have you updated all the tables to add the new columns?

Reply:
Yeah. I have updated all the Perf_Network_TCP table in database.